packages/vite-plugin-cloudflare/README.md

```diff
@@ -1,560 +1,24 @@
 # `@cloudflare/vite-plugin`
 
-[Intro](#intro) | [Quick start](#quick-start) | [Tutorial](#tutorial) | [API](#api) | [Cloudflare environments](#cloudflare-environments) | [Debugging](#debugging) | [Migrating from `wrangler dev`](#migrating-from-wrangler-dev)
+The Cloudflare Vite plugin enables a full-featured integration between [Vite](https://vite.dev/) and the [Workers runtime](https://developers.cloudflare.com/workers/runtime-apis/).
+Your Worker code runs inside [workerd](https://github.com/cloudflare/workerd), matching the production behavior as closely as possible and providing confidence as you develop and deploy your applications.
 
-## Intro
+## Documentation
 
-The Cloudflare Vite plugin enables a full-featured integration between Vite and the Workers runtime.
-Your Worker code runs inside [workerd](https://github.com/cloudflare/workerd), matching the production behavior as closely as possible and providing confidence as you develop and deploy your applications.
+Full documentation can be found [here](https://developers.cloudflare.com/workers/vite-plugin/).
 
-### Features
+## Features
 
-- Provides direct access to Workers runtime APIs and bindings
-- Supports Workers Assets, enabling you to build static sites, SPAs, and full-stack applications
+- Uses the Vite [Environment API](https://vite.dev/guide/api-environment) to integrate Vite with the Workers runtime
+- Provides direct access to [Workers runtime APIs](https://developers.cloudflare.com/workers/runtime-apis/) and [bindings](https://developers.cloudflare.com/workers/runtime-apis/bindings/)
+- Builds your front-end assets for deployment to Cloudflare, enabling you to build static sites, SPAs, and full-stack applications
+- Official support for [React Router v7](https://reactrouter.com/) with server-side rendering
 - Leverages Vite's hot module replacement for consistently fast updates
 - Supports `vite preview` for previewing your build output in the Workers runtime prior to deployment
 
-## Quick start
```

Comment:
I feel like it might be kinda nice to keep a simple quick start here?

Reply:
I could copy over the 'Get started' guide - https://developers.cloudflare.com/workers/vite-plugin/get-started/. What do you think?

Reply:
I would avoid adding much. Perhaps just what the config looks like:

```
import { defineConfig } from "vite";
import { cloudflare } from "@cloudflare/vite-plugin";

export default defineConfig({
  plugins: [cloudflare()],
});
```

Reply:
[26d698b](https://github.com/cloudflare/workers-sdk/pull/8787/commits/26d698bc4d4948b2b6d5377d6ef0cfcdb8f0fba9)